### DISCUSSION TITLE
Are we able to access api calls in the readme?

### DISCUSSION THREAD

**Discussion by MojoJojo86** [renovatebot/renovate#23293] [Suggest an Idea]
### Tell us more.

We don't have github tests enabled in our enterprise hosted version of github.

This means we are obviously going to running the risk of auto-merging with renovate in this situation and have already encountered issues where out CI pipeline has failed but we have continued to automerge additional pr's on top of this.

We do however populate our project readme's with these badges that are generated from API calls to our CD/CI pipelines.
<img width="138" alt="Screenshot 2023-07-11 at 10 08 17" src="https://github.com/renovatebot/renovate/assets/15667195/a0df9e5c-ef7a-4d72-bcfa-d9606fccf035">
Im wondering if its possible for me to write something in renovate that will let you access this value and then make that merge decision based on whether this is a pass or fail.

**Comment by rarkins** [comment]
> We don't have github tests enabled in our enterprise hosted version of github.

No tests in any repo for your whole GitHub Enterprise Server?

> This means we are obviously going to running the risk of auto-merging with renovate in this situation and have already encountered issues where out CI pipeline has failed but we have continued to automerge additional pr's on top of this.

Yes, we recommend you never automerge without tests (although by default Renovate shouldn't automerge unless there's at least one passing test)

> We do however populate our project readme's with these badges that are generated from API calls to our CD/CI pipelines. <img alt="Screenshot 2023-07-11 at 10 08 17" width="138" src="https://user-images.githubusercontent.com/15667195/252614021-a0df9e5c-ef7a-4d72-bcfa-d9606fccf035.png"> Im wondering if its possible for me to write something in renovate that will let you access this value and then make that merge decision based on whether this is a pass or fail.

No. The merge decision is based on whether there's status checks or not
